scripts/hyperkit.mjs

```diff
@@ -149,15 +149,15 @@ export default async function run() {
   await download(
     'https://github.com/jandubois/tinyk3s/raw/v0.1/run-k3s',
     path.resolve(process.cwd(), 'resources', os.platform(), 'run-k3s'),
-    false, fs.constants.X_OK);
+    '', false, fs.constants.X_OK);
```

Comment:
The last three arguments are all defaults; would it make sense to just remove them? (Same with the next two hunks, plus `wsl.mjs`.)

Reply:
Thought of it, good idea.